### ISSUE TITLE
Strange errors in output window

### ISSUE THREAD

**Issue by hitman-codehq** [microsoft/vscode-makefile-tools#423]
Recently, I noticed that the following errors are output many times over in the output window of Visual Studio Code:

```
Error: The task "makefile: Makefile Tools Build Task" has neither a command nor a dependsOn-property. The task will be ignored. Associated definition:
{
    "type": "shell",
    "id": "shell,c:\\amiga-gcc\\bin\\make.exe,all,-j4,DEBUG=1,",
    "problemMatcher": [
        "$gcc",
        "$unknown"
    ],
    "label": "makefile: Makefile Tools Build Task"
}

The problem matcher "{0}" can not be resolved. The matcher will be ignored.
```

I had to translate this manually from German, so the text doesn't match exactly, sorry.

I don't really know where to start with this warning or what it means.  The makefile plugin is working perfectly, including all commands (clean, build), Intellisense scanning and the problems tab.  So it's not causing problems using the pluging, but I thought you might want to know about it.

Many thanks.

**Comment by andreeis** [comment]
@hitman-codehq , thank you very much. Other users reported this error message and I finally reproduced it on my end. I also see how all else works fine and I found some references on the VSCode team about a bogus similar message that is wrongly reported for healthy written tasks. I will follow up with the vscode team, probably a new bug or regression in their code base.
I'll keep your bug report open until I know more instead of resolving as duplicate since the other reports were also tracking other issues as well.
